src/ruby/yast/wfm.rb

```diff
@@ -4,6 +4,9 @@
 require "yast/debugger"
 require "yast/profiler"
 require "yast/yast"
+require "cgi"
+
+include CGI::Escape
```

Comment:
This would put `escapeHTML` into the global namespace. it is better to limit it to Yast::WFM.

`include` works for instance methods, `extend` for class methods

```rb
module Yast
  module WFM
    extend CGI::Escape
```

Reply:
I would not mess global namespace and neither WFM namespace and simply just call full `CGI.escape_html` where it is needed.

Reply:
@jreidinger thanks! I did not find that in the unofficial docs :( https://www.rubydoc.info/stdlib/cgi

 `escape_html` has nicer spelling but is not documented in the official docs: https://ruby-doc.org/stdlib-2.7.2/libdoc/cgi/rdoc/CGI.html